exercise/shallow.js

```diff
@@ -0,0 +1,51 @@
+/*
+## shallow
+
+### Instructions
+
+Create a function called `shallow` that allows you to copy the values of all properties from one or more
+objects to a target object.
+
+### Notions
+
+- https://devdocs.io/javascript/global_objects/object/assign
```

Comment:
I think this exercise is too easy.

Reply:
maybe i could merge this exercise with replicate and create a new exercise.

maybe soup letter:
```js
    const grid = [
      'jefblpepre',
      'camdcimgtc',
      'oivokprjsm',
      'phelloroua',
      'rixieelhrs',
      'wolcqereht',
      'screeaumgr',
      'alxhpburyi',
      'jalaycalmp',
      'cloperermt',
    ]
// result
{
  hello: {
    start: [3, 1],
    end: [3, 6],
  },
  there: {
    start: [5, 10],
    end: [5, 6],
  },
}
```
let me now what do you think

Reply:
yeah ok, it's more an algorithmic exercise than a simple data shape thing, but it has some object output, could be interesting.
I just feel like it's fairly hard to do it with the `map`, `filter`, i'll try.